README.md

```diff
@@ -1,101 +1,256 @@
+- [Elyra][5], which is a set of AI-centric extensions to JupyterLab Notebooks (e.g. interface with Kubeflow Pipeline, Git, Python scripts).
 
-For the purpose of this tutorial we focus on local use of [Elyra][2].
+([WIP](https://github.com/thoth-station/jupyterlab-requirements/issues/53)) The use of the JupyterLab notebook is powered by Thoth Extension for Dependency Management on JupyterLab.
+If you want to know more meanwhile, have a look at this [video](https://www.youtube.com/watch?v=IBzTOP4TCdA).
 
 ## GitOps, reproducibility, portability and traceability with AI support
 
-Nowadays, developers (including Data Scientists) use Git and GitOps practices to store, share all notebooks, sources on development platforms (e.g. GitHub).
-GitOps best practices help reproducibility and traecability for all projects avaialble. One of the most important requirement for reproducibility is dependencies management.
+Nowadays, developers (including Data Scientists) use Git and GitOps practices to store, share all code (e.g. notebooks, models) on development platforms (e.g. GitHub).
+GitOps best practices help reproducibility and traecability for all projects avaialble.
 
-Having dependencies clearly stated allow for reusability and portability of notebooks, which can be reused in another projects.
+One of the most important requirement for reproducibility is dependencies management. Having dependencies clearly stated allow for reusability and portability of notebooks,
+which can be reused in another projects and shared safely with other Data Scientists.
 
-[Project Thoth][1] helps developers keep these depencies up to date, moreover improve developers work creating software stacks for their project that satisfies developers requirements.
-These requirments might across the AIDevSecOps lifecycle of a project, therefore the software stack requirements can change as well.
+(WIP) If you want to know more about this issue in the data science domain, have a look at this [article](https://github.com/thoth-station/jupyterlab-requirements).
 
-For the purpose of this tutorial the type of recommendations requested are different depending on the step of the pipeline required. See [.thoth.yaml file](https://github.com/thoth-station/elyra-aidevsecops-tutorial/blob/master/.thoth.yaml).
+[Project Thoth][6] helps developers keep these dependencies up to date integrating its recommendation in developers daily tools. If you want to know more have a look [here](https://thoth-station.ninja/docs/developers/adviser/integration.html).
+Thanks to these tooling, the developers (including data scientists) do not have to worry about dependency management, which can be handled by bot and automated pipelines.
+Moreover having AI support can lead to improvement in the developer of AI project, speeding up steps due to performance improvements or avoiding security issues due to the introduction
+of libraries that should not be used production.
```

Comment:
....of libraries that should not be used production.

this sounds to me as if you are saying the opposite to what you wanted to say
avoid issues by using libs that should not be used